Assets/Scripts/ScoreManager.cs

```diff
@@ -15,6 +15,7 @@ public class ScoreData
         public DateTime date;
     }
 
+    public const int maxNameLength = 12;
     public const int maxHighscores = 10; // Is 10 highscores enough ?
```

Comment:
should we serialize these? or make them private?

Reply:
They should be private. We don't expect anything to use them.

Reply:
Oh yes I agree this should be private

Reply:
I put them public so if any code needs to use thoses values it can
(for instance, when the player will input his name, we could play a sound if he tries to input more characters than the max size, in this case we will need to use the `maxNameLength` )

Since they are constant, no external code can modify it, I don't see why there is any issue to let them public ?

Reply:
i think the standard is to make it private and make public getter functions. honestly, I don't mind since I know we are the only ones who will ever touch this code. I will approve